testscript/doc.go

```diff
@@ -166,6 +166,11 @@ The predefined commands are:
 - mkdir path...
   Create the listed directories, if they do not already exists.
 
+- unquote file...
+  Rewrite each file by replacing any leading ">" characters from
```

Comment:
I wonder whether it's more intuitive/useful to reference the txtar package docs here. Because the method by which we quote is defined there, as is the unquote?

Reply:
I'm not sure. I think it's nice if the reader of these docs (probably someone that's trying to understand a test failure) doesn't need to go another level deeper. I'll add a note though.